src/components/ResultCard.tsx

```diff
@@ -37,20 +43,46 @@ const ResultCard = memo(
               onChange={() => onCheckboxChange(datasetUUID)}
             />
           </div>
-          <div className="col-span-10 col-start-1">
+          <div className="col-span-6 col-start-1">
             <Typography variant="h5">{datasetName}</Typography>
             <Typography variant="subtitle1">from {nodeName}</Typography>
             <Typography variant="subtitle2">
               {numMatchingSubjects} subjects match / {datasetTotalSubjects} total subjects
             </Typography>
           </div>
-          <div className="col-span-2 justify-self-end">
+          <div className="col-span-2 col-start-7">
+            <Tooltip
+              data-cy={`card-${datasetUUID}-available-pipelines-tooltip`}
+              title={
+                <Typography variant="body1">
+                  {Object.entries(pipelines)
+                    .flatMap(([name, versions]) =>
+                      versions.map((version) => `${name.slice(65)} ${version}`)
+                    )
+                    .map((pipeline) => (
+                      <Divider>{pipeline}</Divider>
+                    ))}
+                </Typography>
```

Comment:
another section for unwrapping pipeline names into text that (on first glance)
shares quite a lot of overlap with code in `ResultContainer.tsx` and might be 
able to share a generic function, also to improve readability here!

Reply:
I don't think this one could use the same generic function used `ResultContainer.tsx` as it involves different logic.